### PR TITLE
Update Nightfall Config detectors to be array

### PR DESCRIPTION
**Pull request by henkejosh**
The API doesn't currently support detector thresholds, so I'm updating the way we handle detector configuration to reflect that.

### CODE REVIEW

File: .github/workflows/nightfalldlp.yml [hunk 1]
```diff
@@ -58,4 +57,4 @@ jobs:
           NIGHTFALL_API_KEY: ${{ secrets.NIGHTFALL_API_KEY }}
           EVENT_BEFORE: ${{ github.event.before }}
         run: |
-          nightfalldlp
+          nightfalldlp --debug
```
Comment (bradleymcallister97) [inline .github/workflows/nightfalldlp.yml]: I do not believe the flag will actually do anything for the Github Action

File: internal/clients/diffreviewer/github/github_service.go [hunk 2]
```diff
@@ -177,11 +177,12 @@ func (s *Service) LoadConfig(nightfallConfigFileName string) (*nightfallconfig.C
 	if s.CheckRequest.SHA == "" {
 		s.CheckRequest.SHA = event.HeadCommit.ID
 	}
-	nightfallConfig, err := nightfallconfig.GetConfigFile(workspacePath, nightfallConfigFileName)
+	nightfallConfig, err := nightfallconfig.GetNightfallConfigFile(workspacePath, nightfallConfigFileName)
 	if err != nil {
 		s.Logger.Error("Error getting Nightfall config file. Ensure you have a Nightfall config file located in the root of your repository at .nightfalldlp/config.json with at least one Detector enabled")
 		return nil, err
 	}
+	s.Logger.Debug(fmt.Sprintf("num detectors: %d %s", len(nightfallConfig.Detectors), *nightfallConfig.Detectors[0]))
```
Comment (bradleymcallister97) [inline internal/clients/diffreviewer/github/github_service.go]: How come we are logging the first Detector (`*nightfallConfig.Detectors[0]`)?

File: internal/clients/nightfall/nightfall_internal_test.go [hunk 3]
```diff
@@ -5,8 +5,9 @@ import (
 	"testing"
 
 	"github.com/nightfallai/nightfall_cli/internal/clients/diffreviewer"
+
 	githublogger "github.com/nightfallai/nightfall_cli/internal/clients/logger/github_logger"
-	"github.com/nightfallai/nightfall_cli/internal/nightfallconfig"
+
```
Comment (bradleymcallister97) [inline internal/clients/nightfall/nightfall_internal_test.go]: nit - do we need these line breaks? Or can we just put them in one "paragraph"

**Review by bradleymcallister97 — APPROVED** [review]

File: internal/clients/nightfall/nightfall_internal_test.go (hunk 3 above)
Comment (alan20854) [reply internal/clients/nightfall/nightfall_internal_test.go]: yea we typically want them in a paragraph, but goland sometimes adds line breaks for new imports that are added

**Review by alan20854 — APPROVED** [review]

**Pull request merged**